README.md

```diff
@@ -71,16 +71,21 @@ Usage: peodd [OPTIONS]
   Script to export the pyproject.toml dev-dependencies to a txt file.
 
 Options:
-  -o, --output TEXT  Output file for the dependencies  [default: requirements-
-                     dev.txt]
+  -o, --output TEXT  Output file for the dependencies  [default:
+                     requirements.txt]
```

Comment:
Since we removed the default, I think it should be updated. I have a hack for this.

```
$ poetry install
$ poetry shell
(.venv) $ peodd --help
```

Copy this content and paste it here.

Reply:
Usage: peodd [OPTIONS]

  Script to export the pyproject.toml dev-dependencies to a txt file.

Options:
  -o, --output TEXT  Output filename for the dependencies
  --non-dev          Output non-dev dependencies
  --help             Show this message and exit.